style.css

```diff
@@ -419,6 +420,68 @@ textarea::placeholder {
   color: var(--white-color);
 }
 
+/* =========== modal css ======== */
+.modal {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  transition: 200ms ease-in-out;
+  border: 1px solid var(--line-color);
+  background-color: var(--white-color);
+  z-index: 50;
+  width: 500px;
+  max-width: 80%;
+  padding: 1rem;
+  border-radius: 25px;
+}
+
+.modal.actives {
+  transform: translate(-50%, -50%) scale(1);
+}
+
+.modal-title {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.close-modal-btn {
+  cursor: pointer;
+  border: none;
+  outline: none;
+  background: none;
+  font-size: 1.25rem;
+  font-weight: bold;
+  transition: color 0.3s ease-out;
+}
+
+.close-modal-btn:hover {
+  color: var(--secondary-color);
+}
```

Comment:
## Kindly adjust the requirement in the mobile version as well 💪🏻💪🏻
- [ ] As I can see here when I hover outside the card it closes but I think you didn't add the X icon to let the user  ba able to close the modal whenever he want💪🏻💪🏻 so kindly add it to the modal ✅✅
![image](https://user-images.githubusercontent.com/58553711/180231984-c1fd76c7-e191-4afc-bdc3-7db260e80464.png)

Reply:
@Reem-lab Thank you for the review, All the changes have been made, together with my coding partner. 
Kindly check and let me know if there is anything that may be required.